qa/tasks/scripts/install_migration_rpm.sh

```diff
@@ -22,4 +22,11 @@ function zypper_install {
 
 zypper_install $RPM
 zypper_install suse-migration-sle15-activation
+
+SLE_MIGRATION_YML="/etc/sle-migration-service.yml"
+cat <<EOM > $SLE_MIGRATION_YML
+use_zypper_migration_plugin: false
```

Comment:
Needs to be changed to `use_zypper_migration: false` once we have access to the newer version of the migration RPM (SLES15-Migration-1.15.2.ses.0-15.x86_64.rpm)

Reply:
@tserong Unknown yaml keys will be ignored, right? In that case, I could do:

```
use_zypper_migration_plugin: false
use_zypper_migration: false
```

and it will be compatible with both 1.15.1 and 1.15.2 - correct?

Reply:
Yeah, having just had a quick look at the code that should be safe.